lib/video_cut_and_merge.ex

```diff
@@ -54,25 +54,25 @@ defmodule Membrane.VideoCutAndMerge do
   end
 
   @impl true
-  def handle_init(_opts) do
-    children = [{:merger, VideoMerger}]
-    links = [link(:merger) |> to_bin_output]
-    spec = %ParentSpec{children: children, links: links}
+  def handle_init(_ctx, _opts) do
+    structure = child(:merger, VideoMerger) |> bin_output
 
-    {{:ok, spec: spec}, nil}
+    {[spec: structure], nil}
   end
 
+  # TODO Remove this when we have fix in Membrane Core
```

Comment:
```suggestion
  # TODO: Remove this when we have fix in Membrane Core
```
Nitpick, but with a colon it's highlighted nicely in VSC :smiley: